common/CommonAPI.h

```diff
@@ -1,20 +1,29 @@
 #ifndef __NBL_COMMON_API_H_INCLUDED__
 #define __NBL_COMMON_API_H_INCLUDED__
 
+#define _NBL_STATIC_LIB_
```

Comment:
can you live without this?

Reply:
don't remember what that is, but when I remove it, it still works :D

Reply:
remove it, its a define that should only be defined when CMake is in the static config, shoul dbe already in `BuildConfig.h` which is autogeneretated

Reply:
I believe a lot of examples were defining it, may be a remnant of very old code. `_NBL_SHARED_BUILD_` is defined instead https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/config/BuildConfigOptions.h.in#L90